oca_dependencies.txt

```diff
@@ -0,0 +1,2 @@
+# See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
+stock-logistics-workflow
```

Comment:
But that one is merged

Reply:
Thanks @rousseldenis I did not know oca_dependencies was deprecated.
So, in case all dependencies are already merged, I have nothing  to do.

And if I don't care the PR beeing green while the dependencies are not merged, I guess the test-requirements.txt is optional and should be removed before merged right ?